suite-native/transactions/src/components/TransactionsList/TransactionIcon.tsx

```diff
@@ -52,8 +62,21 @@ const cryptoIconStyle = prepareNativeStyle<TransactionIconStyleProps>(
     }),
 );
 
+const CoinIcon = ({ symbol }: { symbol: NetworkSymbol | EthereumTokenSymbol }) => {
+    if (symbol in networks) {
+        return <CryptoIcon name={symbol as NetworkSymbol} size={COIN_ICON_SIZE} />;
+    }
+
+    return (
+        <EthereumTokenIcon
+            name={(symbol in ethereumTokenIcons ? symbol : 'erc20') as EthereumTokenIconName}
```

Comment:
this is getting duplicated. Please create type predicate in ethereum tokens package

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25